decidim-budgets/app/cells/decidim/budgets/groups/base_cell.rb

```diff
@@ -9,7 +9,19 @@ class BaseCell < Decidim::ViewModel
         include Decidim::ComponentPathHelper
         include Decidim::Budgets::Engine.routes.url_helpers
 
-        delegate :current_user, to: :controller
+        delegate :current_user, :parent_component_context, to: :controller
+        delegate :current_settings, :settings, to: :group_component
+
+        alias component model
+        alias group_component model
+
+        def workflow_instance
+          @workflow_instance ||= parent_component_context[:workflow_instance]
+        end
+
+        def limit_reached?
+          current_user && (workflow_instance.allowed - workflow_instance.progress).none?
```

Comment:
should this also include the `voted`?
```suggestion
          current_user && (workflow_instance.allowed - [workflow_instance.progress + workflow_instance.voted]).none?
```

Reply:
yes, I think you are right